docs/source/pages/howToWorkWithPlatform/mainApplication/scenario/scenarioManagement/scenarioManagement.rst

```diff
@@ -282,3 +282,40 @@ OR
             ...
         </xsl:template>
     </xsl:stylesheet>
+
+.. _run-scenario-from-integration:
+
+Run scenario from integration
+=============================
+
+Customer is able to run specific scenario through integration functionality. As soon as customer creates scenario, it possible to run through integration, to do that customer should define scenario UUID inside envelope XML in integration rule.
+
+.. note:: Integration data has highest priority and will replace all scenario data if it's defined in integration rule
+
+Defined scenario UUID should be inside <envelope> tag as attribute scenarioUuid="uuidOfScenario"
+
+.. code:: xml
+
+    <?xml version="1.0" encoding="UTF-8"?>
+    <xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+        <xsl:template match="/">
+            <envelope templateUuid="19089ab8-a828-4447-9129-44b1da1cbc9d" templateVersion="f2e1a1c5-15ac-4bfb-bf31-f1ddba652b89" scenarioUuid="e17db760-580a-4192-bc96-a91683123bea">
+	            ...
+            </envelope>
+        </xsl:template>
+    </xsl:stylesheet>
+
+.. note:: You may run scenario from any step you want through integration. System automatically define first step which match template and version and create envelope using data you've provided in integration rule.
+
+Scenario configuration of chaining envelope will be ignored by default if customers create envelope with defined scenario inside integration rule. But customer is able to define chain UUID inside <envelope> tag as attribute chainUuid="uuidOfChain". If user wants to chain all created envelopes though integration it could be static UUID which should be defined once in integration rule, but if customer wants to chain envelopes which created for each file pulled though integration separately it should be defined dynamically.
```

Comment:
- Scenario configuration of chaining envelope**S**
- ...if customer create**S**
- ...but if customer wants to chain envelopes which created for each... **word which is redundant**